defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 # defaults file for arillso.logrotate
 
+use_hourly_rotation: false
```

Comment:
To make everything consistent, the `logrotate` role prefix should be used.

Reply:
Fixed in 17cc47173ce9945c74dd3629f6912df504a8eeeb